run_codenarc.py

```diff
@@ -357,6 +370,15 @@ def run_codenarc(args, report_file=DEFAULT_REPORT_FILE):
         args.log_level
     ]
 
+    if args.single_file:
+        extra_args = [f'-includes={args.single_file}']
```

Comment:
```suggestion
        extra_args = [f'-includes={os.path.basename(args.single_file)}']
```
Since `args.single_file` might be a composite path.